drivers/ethernet/eth_stm32_hal.c

```diff
@@ -32,19 +33,67 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #error DTCM for DMA buffer is activated but zephyr,dtcm is not present in dts
 #endif
 
+#ifndef PHY_ADDR
```

Comment:
A separate layer to PHY SoC would be more clear way to implement ETH_PHY layer. This value actually belongs there. I will just delete `#ifndef`. Thus it will follow the original implementation in current master.

Reply:
Corresponding `#ifndef` will be removed within next push.

Reply:
There is no other place. Removed corresponding `#ifndef`.